analyses/molecular-subtyping-HGG/02-HGG-molecular-subtyping-subset-files.R

```diff
@@ -105,37 +128,41 @@ hgg_lesions_df <- hgg_lesions_df %>%
 #### Filter metadata -----------------------------------------------------------
 
 # Filter metadata for `High-grade glioma` and samples that should be classified
-# as High-grade glioma based on defining lesions and are stranded RNA-seq
-# samples.
+# as High-grade glioma based on defining lesions
 hgg_metadata_df <- metadata %>%
   dplyr::filter(
     disease_type_new == "High-grade glioma" |
       sample_id %in% hgg_lesions_df$sample_id,
-    experimental_strategy == "RNA-Seq",
-    RNA_library == "stranded"
+    sample_type == "Tumor",
```

Comment:
This filter is what gets rid of cell line derived samples?

Reply:
Yep - the composition step, specifically.